ts/src/kucoin.ts

```diff
@@ -854,7 +854,7 @@ export default class kucoin extends Exchange {
         const networkCode = this.safeStringUpper (params, 'network');
         const network = this.networkCodeToId (networkCode, code);
         if (network !== undefined) {
-            request['chain'] = network;
+            request['chain'] = network.toLowerCase ();
```

Comment:
> ETH, eth does not work, only ERC20 can work here.

Hey @leastchaos, if so this won't fix it right?

Reply:
I think the chain is expecting a lower case value. it is not possible to be lower case here unless the network entered is inside the networkCode.
I tried ETH and eth. both will remain as ETH when passed into the request['chain']. and it will return invalid chainId
but ERC20 will be converted to eth here and then it will work.